browser/resources/settings/brave_clear_browsing_data_dialog/brave_clear_browsing_data_on_exit_page.html

```diff
@@ -28,6 +28,14 @@
                       sub-label="[[counters.cookies]]"
                       no-set-pref>
   </settings-checkbox>
+  <settings-checkbox id="leoResetOnExitCheckbox"
+                      class="cache-checkbox"
```

Comment:
nit: I don't think this `class` is neccessary - it doesn't seem to do anything

Reply:
removed